gluestack-ui.config.ts

```diff
@@ -23,6 +25,49 @@ export const config = {
       rounded: "borderRadius",
     } as const,
     tokens: {
+      colors: {
+        primary50: "#f0f9ff",
+        primary100: "#e0f2fe",
+        primary200: "#bae6fd",
+        primary300: "#7dd3fc",
+        primary400: "#38bdf8",
+        primary500: "#0ea5e9",
+        primary600: "#0284c7",
+        primary700: "#0369a1",
+        primary800: "#075985",
+        primary900: "#0c4a6e",
```

Comment:
I assume these are autogenerated but do we need them all?

Reply:
Not all of them. I'll try to clean out the ones we don't use.

Reply:
We should change to completely remove using these variables but you don't have to do that here.

Reply:
Yeah, we used some random colors from Native Base so I wanted that change to be deliberate.